frontend/src/strings/crawl-workflows/labels.ts

```diff
@@ -7,4 +7,5 @@ export const labelFor = {
   postLoadDelaySeconds: msg("Delay After Page Load"),
   behaviorTimeoutSeconds: "Behavior Limit",
   pageExtraDelaySeconds: msg("Delay Before Next Page"),
+  selectLink: msg("Custom Link Selectors"),
```

Comment:
I wonder if this should just be "Link Selectors". Two reasons:
- The table is already populated with the default link selector for a new workflow
- I think this would be more consistent with how we've named other fields, e.g. "User Agent", which was initially named "Custom User Agent" before we decided to drop the "Custom"